lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -83,26 +86,36 @@ public Task claim(String id, String userName) throws TaskNotFoundException {
     }
 
     @Override
-    public Task complete(String id) throws TaskNotFoundException {
-        LOGGER.debug("entry to complete(id = {})", id);
+    public Task completeTask(String taskId, boolean isForced) throws TaskNotClaimedException, TaskNotFoundException, NotOwnerException {
```

Comment:
Should be seperated into **_completeTask(String taskId)_** and **_completeTask(String taskId, boolean isForced)_**

**_completeTask(String taskId)_** just needs to call **_completeTask( , )_** with false

Reply:
Done.